tests/test_simple_api.py

```diff
@@ -27,13 +27,17 @@ def test_simple_api(alices_keys, N, M, curve=default_curve()):
     plain_data = b'peace at dawn'
     ciphertext, capsule = pre.encrypt(delegating_privkey.get_pubkey(), plain_data, params=params)
 
+    capsule.set_correctness_keys(delegating=delegating_privkey.get_pubkey(),
+                     encrypting=decrypting_key.get_pubkey(),
+                     verifying=signing_privkey.get_pubkey())
+
     cleartext = pre.decrypt(ciphertext, capsule, delegating_privkey, params=params)
     assert cleartext == plain_data
 
     kfrags = pre.split_rekey(delegating_privkey, signer, decrypting_key.get_pubkey(), M, N, params=params)
     for kfrag in kfrags:
         cfrag = pre.reencrypt(kfrag, capsule, params=params)
-        capsule.attach_cfrag(cfrag)
+        capsule.attach_cfrag(cfrag, params=params)
```

Comment:
Maybe not for this PR, but.. Why do we have to carry the params? Cannot the params be set when the capsule is originally instantiated?

Reply:
Yeah, that occurred to me.  There's... probably no good reason.  When I started working on this PR, I was in a way of trying to be sort of minimally invasive with respect to these sorts of changes, but that appears to have been a bit misguided.  :-)